application/core/LSYii_Application.php

```diff
@@ -56,6 +56,9 @@ class LSYii_Application extends CWebApplication
      */
     protected $dbVersion;
 
+    /* @var integer| null the current userId for all action */
+    private $currentUserId;
```

Comment:
Here or on the trait (which maybe not here yet because of other story not merged yet)?

Reply:
I do,'t understand your question ? There are no trait for LSYii_Application

Reply:
OK, got it … think it resolved now ?

Reply:
This bheaviour may be needed in both console application and web application, so that's why I am saying to add it to a trait. We created a trait for 18961. Still that is not merged. That was what I was referring to.